File: src/main/java/com/github/eendroroy/sdk/openweather/response/WindResponse.java
package com.github.eendroroy.sdk.openweather.response;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonRootName;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

/**
 * @author indrajit
 */

@ToString
@Setter
@Getter
@JsonRootName("wind")
public class WindResponse implements BaseResponse {
    @JsonProperty("speed")
    public Double speed;

    @JsonProperty("deg")
    public Integer deg;
}

File: src/main/kotlin/com/github/eendroroy/sdk/openweathermap/response/WindResponse.kt
package com.github.eendroroy.sdk.openweathermap.response

import com.fasterxml.jackson.annotation.JsonProperty
import com.fasterxml.jackson.annotation.JsonRootName

/**
 * @author indrajit
 */
@JsonRootName("wind")
class WindResponse : BaseResponse {
    @JsonProperty("speed")
    var speed: Double? = null

    @JsonProperty("deg")
    var deg: Int? = null
}